app/models/review_model.py

```diff
@@ -0,0 +1,28 @@
+import uuid
+from pydantic import BaseModel, Field, validator
+from typing import Optional, List
+from datetime import datetime
+
+class ReviewModel(BaseModel):
+    id: str = Field(default_factory=uuid.uuid4, alias="_id")
+    username: Optional[str] = Field(None)
```

Comment:
Same thing happens in Recipe, when we POST a recipe we don't add the username in the JSON, but instead we use the user token to later in the recipe_router identify its user and add it to the recipe, so it will always have a username.